CHANGELOG.md

```diff
@@ -78,9 +78,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
   - Change the target of tag expressions [(#15715)](https://github.com/ManageIQ/manageiq/pull/15715)
   - MiqExpression::Target#to_s [(#15713)](https://github.com/ManageIQ/manageiq/pull/15713)
   - Rename applies_to_exp to visibility_expression for serializing [(#15501)](https://github.com/ManageIQ/manageiq/pull/15501)
-- Workers
-  - [Rearch] Combine worker messages 'sync_config' and sync_active_role' into a single 'sync_config' message. [(#15597)](https://github.com/ManageIQ/manageiq/pull/15597)
-  - Add server MB usage to rake evm:status and status_full. [(#15457)](https://github.com/ManageIQ/manageiq/pull/15457)
+  - Workers
+    - [Rearch] Combine worker messages 'sync_config' and sync_active_role' into a single 'sync_config' message. [(#15597)](https://github.com/ManageIQ/manageiq/pull/15597)
```

Comment:
I didn't notice before but the rearch tag makes no sense here.  If it was committed to master, it's not rearch specific anymore